src/main/java/WayofTime/bloodmagic/util/helper/RitualHelper.java

```diff
@@ -117,4 +119,135 @@ else if (tile != null && tile.hasCapability(RUNE_CAPABILITY, null)) {
             world.notifyBlockUpdate(pos, state, state, 3);
         }
     }
+
+    public static void removeValidRituals(TileMasterRitualStone tile) {
```

Comment:
What even is this method? The MRS knows what ritual it has active, use that.

Reply:
The MRS knows what ritual it has but this can remove remains from a non-activated ritual (i.e. something is in the way when using the Ritual Diviner) and it can remove rituals that are complete but have not been activated.